tutorials/quick_start/pytorch_fw/ultralytics_lib/replacers.py

```diff
@@ -22,49 +22,41 @@
 import torch
 
 from common.model_lib import ModuleReplacer
-from ultralytics import YOLO, yolo
-from ultralytics.nn.modules import C2f, Detect, Segment
-from ultralytics.nn.tasks import DetectionModel, ClassificationModel, SegmentationModel
-from ultralytics.yolo.cfg import get_cfg
-from ultralytics.yolo.utils import DEFAULT_CFG
-from ultralytics.yolo.utils.checks import check_imgsz
-from ultralytics.yolo.v8.detect import DetectionValidator
+from overrides import override
```

Comment:
is this mandatory?
If I'm not mistaken, this import forces MCT's requirements to include the overrides package (at least for using the tutorials package inside MCT), so either remove it or add it to the requirements (not sure if we want to do this).

Reply:
We use this to make sure we override an existing function in the ultralytics package. Without it if they change their code (function signature for example) we will fail without realizing our implementation isn't used

Reply:
Ok. so if it is required to run the tutorials I think that it should be added to the requirements file (or maybe there should be a separate requirements file for the quick start project? because this package is not mandatory to run MCT).

Reply:
removed override decorator :(